packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/index.tsx

```diff
@@ -205,11 +205,10 @@ export const GitHub = () => {
         <Collapsible title="GitHub Repository" defaultOpen>
           <Element paddingX={2}>
             <Text variant="muted">
-              If you wish to contribute back to{' '}
-              {forkedTemplateSandbox.git.username}/
-              {forkedTemplateSandbox.git.repo}, you can link this sandbox to the
-              git repository. This will allow you to create commits and open
-              pull requests with this sandbox.
+              If you wish to contribute back to {upstreamSandbox.git.username}/
```

Comment:
Better context:

![image](https://user-images.githubusercontent.com/587016/88065032-5a4b1d00-cb6c-11ea-81f6-d7b005113900.png)

there's a check here on `upstreamSandbox?.git`, which we weren't using in the text.